cloud/amazon/ec2.py

```diff
@@ -883,6 +883,10 @@ def create_instances(module, ec2, vpc, override_count=None):
                 grp_details = ec2.get_all_security_groups()
             if isinstance(group_name, basestring):
                 group_name = [group_name]
+            unmatched = list(set(group_name) - set([str(grp.name) for grp in grp_details]))
+            if len(unmatched) > 0:
+                module.fail_json(
```

Comment:
wouldn't bother with the line break here - longer but readable lines are preferred over strict PEP-8 line length compliance.